### PR TITLE
feat: added animated Wumpus

### PR DESCRIPTION
**Pull request by kpracuk**
Let me know about any bugs.
A lot has changed from Rive 1 -> 2 both in the editor and in the runtime.

### CODE REVIEW

**Comment by zSoulweaver** [comment]
~~LGTM~~ I lied it don't work ahaha
EDIT: It breaks if another page loads first and then route to index
"RiveAnimation is not defined"


This is now easily my most favourite thing on the site. I'm going to spend too procrastinating and staring at the animation 😂

Played around with a few things when I was having a look, what do you think about rotating it about 3 degrees?

image

**Comment by kpracuk** [comment]
Wumpus is not going rouge anymore. Home page is safe from destruction (for now)
Also, he is now rotated -3 degrees and looks a whole lot cuter.

**Comment by kpracuk** [comment]
Ok, the `fs: 'empty'` fix worked but it was throwing errors later during runtime.
For now I left this on CDN.

File: pages/index/-components/Wumpus.vue [hunk 1]
```diff
@@ -7,23 +7,28 @@ export default {
...
+          hid: 'rive',
+          src: 'https://unpkg.com/rive-js/dist/rive.js',
+          callback: () => {
+            // eslint-disable-next-line no-new, no-undef
+            new RiveAnimation({
+              src: '/rive/wumpus.riv',
+              canvas: this.$refs.riveWumpus,
+              autoplay: true
+            })
+          }
```
Comment (zSoulweaver) [inline pages/index/-components/Wumpus.vue]: Would this be better organised if the callback called a method?

Comment (kpracuk) [reply pages/index/-components/Wumpus.vue]: Yup, definitely.
It looks much cleaner

**Comment by zSoulweaver** [comment]
lgtm, merged

**Pull request merged**